### ISSUE TITLE
3.30  Requirements for Equipment Records

### ISSUE THREAD

**Issue by cpp1l** [openhealths/nationHealth#733]
<img width="1507" height="304" alt="Image" src="https://github.com/user-attachments/assets/6c1d63f6-74b9-460d-9cf1-04f864333657" />

- The accessibility button is blocked and may mislead the user into thinking it does not work.
